component/administrator/components/com_foo/helpers/foo.php

```diff
@@ -31,6 +32,6 @@ class FooHelper
 	 */
 	public function addSubmenu($vName)
 	{
-		JHtmlSidebar::addEntry(Text::_('COM_FOO'), 'index.php?option=com_foo&view=foo', $vName == 'foo');
+		HTMLHelper::_('sidebar.addEntry', Text::_('COM_FOO'), 'index.php?option=com_foo&view=foo', $submenu == 'foo');
```

Comment:
@travisrisner There is another issue :) The `$submenu` should be `$vName` as that is the variable passed in.

Reply:
Thanks for the catch. This should be resolved now.